### ISSUE TITLE
Issue generating type adapters

### ISSUE THREAD

**Issue by Caculuz-cinks** [hivedb/hive#395]
**Question**
I've been trying to generate type adapters, but it doesn't complete the process

**Code sample**
This is the log

[INFO] 42.3s elapsed, 4/20 actions completed.
[INFO] 43.4s elapsed, 4/20 actions completed.
[INFO] 44.5s elapsed, 4/20 actions completed.
[INFO] 45.6s elapsed, 4/20 actions completed.
[INFO] 46.6s elapsed, 4/20 actions completed.
[INFO] 47.7s elapsed, 4/20 actions completed.
[INFO] 48.8s elapsed, 4/20 actions completed.
[INFO] 49.8s elapsed, 4/20 actions completed.
[INFO] 50.9s elapsed, 4/20 actions completed.
[INFO] 52.0s elapsed, 4/20 actions completed.
[INFO] 53.1s elapsed, 4/20 actions completed.
[INFO] 54.2s elapsed, 4/20 actions completed.
[INFO] 55.3s elapsed, 4/20 actions completed.
[WARNING] No actions completed for 15.1s, waiting on:
  - hive_generator:hive_generator on lib/core/FakeData/fake_auth.dart
  - hive_generator:hive_generator on lib/core/FakeData/fake_post.dart
  - hive_generator:hive_generator on lib/core/mixin/validators.dart
  - hive_generator:hive_generator on lib/core/models/draft_model/draft_model.dart
  - hive_generator:hive_generator on lib/core/models/login_info.dart
  .. and 11 more

**Version**
 - Platform: iOS, Android, 
 - Flutter version: [e.g. 1.5.4]
 - Hive version: [e.g. 0.5.0]

**Comment by mccidi** [comment]
#387 might help

**Comment by TheMisir** [comment]
Closed as duplicate of #387. Check [this comment](https://github.com/hivedb/hive/issues/387#issuecomment-663900752) to solve the issue.

**Comment by Caculuz-cinks** [comment]
Ok thanks.
